events/smkmeetup/index.html

```diff
@@ -68,52 +68,29 @@ <h3>Photobooth</h3>
-
-  <h5>Museum/Meetup Admission</h5>
-  <div class="table-wrapper">
-    <table> <tbody>
-        <tr>
-          <td>With Keyboard &amp; RSVP</td>
-          <td>RSVP Required</td>
-        </tr>
-        <tr>
-          <td>Without Keyboard*</td>
-          <td>$16.00</td>
-        </tr>
-      </tbody>
-    </table>
-  </div>
-  <p><sup>* If you RSVP and bring a keyboard, the museum will admit you for free because you are an "exhibitor" of keyboards at the museum for the day. This event is public, so anyone can attend without an RSVP. But without an RSVP and keyboard, museum entry fee or museum membership will be required.</sup></p>
+  <p><strong>Seattle Pacific University</strong><br /><em>Upper Gwinn Commons</em><br />3310 Sixth Avenue West,<br />Seattle, WA 98119</p>
+
+  <p>There will also be inexpensive dorm rooms available to book on Friday and Saturday. If you RSVP, you will be asked if you are interested in a dorm room. We will then contact via email once we finalize the details. Not ready to RSVP? You can still go to the RSVP page and register your email address for future dorm updates.</p>
 
   <a name="parking"></a>
   <hr />
 
   <h3>Parking</h3>
-  <p>The Living Computer Museum has limited parking available in their parking lot. Street parking should be free. There are plenty of paid parking options as well.</p>
-
-  <a href="https://en.parkopedia.com/parking/locations/living_computers:_museum_+_labs_1st_avenue_south_seattle_wa_united_states_c23n8nns4mq/?arriving=201701071400&leaving=201701071700" class="button special fit">Show me parking options</a>
+  <p>There will be plenty of on site parking.</p>
 
   <a name="public-transportation"></a>
   <hr />
 
   <h3>Public Transportation</h3>
 
-  <p>According to the Living Computers website:</p>
-
-  <blockquote>The primary bus routes near Living Computers: Museum + Labs are 21, 594, 132, 106, 50, 102, 590, and 116. You can also use the Metro Transit Trip Planner to find a bus route. If you are taking Central Link light rail, get off at the SODO station. Walk a few blocks west to 1st Avenue S, then north one block to the museum.</blockquote>
-
-  <p>You may also find suggestions for Seahawks games useful, however you will be a few blocks away from the museum if you follow Century Link Field directions completely.</p>
-
-  <a href="https://tripplanner.kingcounty.gov/" class="button special fit">Metro Transit Trip Planner</a>
-  <a href="http://www.seahawks.com/gameday/getting-to-centurylink-field" class="button fit">Seahawks Transit Suggestions</a>
+  <p>Details coming soon...</p>
 
   <a name="food-and-beverages"></a>
   <hr />
 
   <h3>Food and Beverages</h3>
 
-  <p>You are welcome to bring your own food/snacks <strong>but they may only be consumed in the cafe area.</strong></p>
+  <p>Lunch will not be served. There will be a break after the morning presentations for lunch. <strong>SPU has requested that no outside food be brought into the venue.</strong></p>
```

Comment:
We should maybe say something about "lunch suggestions forthcoming", and we can detail that on our "travel guide" page or whatever?